pgcli/packages/parseutils.py

```diff
@@ -161,10 +162,26 @@ def find_prev_keyword(sql):
         return None, ''
 
     parsed = sqlparse.parse(sql)[0]
-    for t in reversed(list(parsed.flatten())):
-        if t.is_keyword or t.value == '(':
-            idx = parsed.token_index(t)
-            text = ''.join(tok.value for tok in parsed.tokens[:idx+1])
+    flattened = list(parsed.flatten())
+
+    trivial_keywords = ('AND', 'OR', 'NOT', 'BETWEEN')
```

Comment:
Let's call this logical_operators. 2 months down the line that list won't seem so trivial. :P